docs/reference/flow-yaml-schema-reference.md

```diff
@@ -0,0 +1,222 @@
+| `<node_name>`        | string   | The name of the node.                                                                                                                                                                                                                                                                                         |                |
+| `default_variant_id` | string   | Default variant id.                                                                                                                                                                                                                                                                                           |                |
+| `variants `          | object   | This dictionary contains all node variations, with the variant id serving as the key and a node definition dictionary as the corresponding value.  Within the node definition dictionary, the key labeled 'node' should contain a variant definition similar to [Nodes](#nodes), excluding the 'name' field.  |                |
+
+
+
+## Examples
+
+Flow examples are available in the [GitHub repository](https://github.com/microsoft/promptflow/tree/main/examples/flows). 
+Below, we provide a web classification flow example.
+
+## YAML: Web classification flow YAML
+
+::::{tab-set}
+:::{tab-item} CLI
+:sync: CLI
+
+```yaml
+inputs:
+  url:
+    type: string
+    default: https://www.microsoft.com/en-us/d/xbox-wireless-controller-stellar-shift-special-edition/94fbjc7h0h6h
+outputs:
+  category:
+    type: string
+    reference: ${convert_to_dict.output.category}
+  evidence:
+    type: string
+    reference: ${convert_to_dict.output.evidence}
+nodes:
+- name: fetch_text_content_from_url
+  type: python
+  source:
+    type: code
+    path: fetch_text_content_from_url.py
+  inputs:
+    url: ${inputs.url}
+- name: summarize_text_content
+  type: llm
+  source:
+    type: code
+    path: summarize_text_content.jinja2
+  inputs:
+    deployment_name: text-davinci-003
+    suffix: ''
+    max_tokens: '128'
+    temperature: '0.2'
+    top_p: '1.0'
+    logprobs: ''
+    echo: 'False'
+    stop: ''
+    presence_penalty: '0'
+    frequency_penalty: '0'
+    best_of: '1'
+    logit_bias: ''
+    text: ${fetch_text_content_from_url.output}
+  provider: AzureOpenAI
+  connection: azure_open_ai_connection
+  api: completion
+  module: promptflow.tools.aoai
+
+  use_variants: true
+- name: prepare_examples
+  type: python
+  source:
+    type: code
+    path: prepare_examples.py
+  inputs: {}
+- name: classify_with_llm
+  type: llm
+  source:
+    type: code
+    path: classify_with_llm.jinja2
+  inputs:
```

Comment:
This is too long, maybe have a shorter one here

Reply:
this example contain may fields to help customer understand yaml schema. Maybe It's better than a shorter one?

Reply:
I will remove the yaml here and just put some sample links here.